js/modules/k6/grpc/client.go

```diff
@@ -84,7 +84,7 @@ func (c *Client) Connect(addr string, params map[string]interface{}) (bool, erro
 
 	p, err := c.parseConnectParams(params)
 	if err != nil {
-		return false, err
+		return false, fmt.Errorf("Connect request defines invalid parameters: %w", err)
```

Comment:
Maybe be more precise, since Connect is a generic one? WDYT?
```suggestion
		return false, fmt.Errorf("grpc.Connect has invalid parameters: %w", err)
```